index.html

```diff
@@ -173,22 +191,27 @@ <h1 class="aboutme">About me</h1>
                 <li>Gitlab</li>
                 <li>Terminal</li>
               </ul>
-            </li>
-          </ul>
+            </div>
+          </div>
         </section>
       </div>
     </div>
     <div class="footer">
```

Comment:
Your footer layout is also breaking for larger screens. Kindly fix it.
| Your design  | Figma design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/33268927/177906816-3d91cce1-5d53-4929-819f-4862e8bfd6bd.png) | ![image](https://user-images.githubusercontent.com/33268927/177906868-86b19162-7a47-4587-9255-3c95d1f41bbd.png) |